pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -157,9 +159,8 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	server.TLSCertFile = options.ServingInfo.ServerCert.CertFile
 	server.TLSPrivateKeyFile = options.ServingInfo.ServerCert.KeyFile
 
-	if value := cmdutil.Env("OPENSHIFT_CONTAINERIZED", ""); len(value) > 0 {
-		server.Containerized = value == "true"
-	}
+	containerized := cmdutil.Env("OPENSHIFT_CONTAINERIZED", "") == "true"
+	server.Containerized = containerized
```

Comment:
Nm, intentional.  Because we haven't loaded flags yet - we don't need to use their default because it makes no sense.